contrib/systems.md/thermostat.md

```diff
@@ -46,23 +81,135 @@ mod THERMOSTAT is
                    => < TIME , TMP , off >
                    if TMP < min + bound = false .
 
-    rl [delaying] : { TIME + 1 , TMP , DM }
-                 => < TIME     , TMP , DM > .
+   crl [delaying] : { TIME     , TMP , DM }
+                 => < TIME - 1 , TMP , DM >
+                 if 0 < TIME = true .
 
     rl [delay-over] : { 0 , TMP , delay(IM) }
                    => < 0 , TMP , IM        > .
 ```
 
+The thermostat `heat-rate` is determined as a function of the thermostat parameters.
+
+```maude
+    op heat-rate : Mode TMP$Ring -> TMP$Ring .
+    ------------------------------------------
+    eq heat-rate(MODE, TMP) = (TMP + source(MODE)) - drain(TMP) .
+```
+
 The following are parameters which must be filled in for your particular thermostat.
 
 ```maude
-   ops min max bound : -> Nat .
-   ----------------------------
+    ops min max bound : -> TMP$Ring .
+    ---------------------------------
 
-   ops time-until : InMode -> Nat .
-   --------------------------------
+    op time-until : InMode -> TIME$Ring .
+    -------------------------------------
 
-    op heat-rate : Mode Nat -> Nat .
-    --------------------------------
+    op source : Mode -> TMP$Ring .
+    ------------------------------
+
+    op drain : TMP$Ring -> TMP$Ring .
+    ---------------------------------
+endm
+```
+
+Thermostat Instantiation
+------------------------
+
+### Instantiation to FVP-INT
+
+ ```maude
+load ../tools/fvp/numbers.maude
+
+view RingInt from UNITAL-ORDERED-RING to FVP-INT-PRED is
+sort RingBool to Bool .
+    sort Ring     to Int  .
+endv
```

Comment:
The indentation is inconsistent for the two lines in the view.

Reply:
Fixed.